src/npr/ont_pipeline.py

```diff
@@ -189,7 +202,16 @@ def read_flowcell_info(config, info_dict, base_path):
                 info_dict['barcode_kit'] = info_dict['kit']
             else:
                 info_dict['barcode_kit'] = 'no_bc'
+
+        # getting software and versions
+        if 'software_versions' in jsondata['protocol_run_info']:
+            info_dict['software'] = jsondata['protocol_run_info']['software_versions']
+            if 'guppy_connected_version' in info_dict['software']:
+                info_dict['software']['Dorado'] = info_dict['software']['guppy_connected_version']
     else:
+        ##################################################################################
```

Comment:
I don't understand this section, is an else here needed ?

Reply:
no, those were some note marks. removed